QUICKSTART.rst

```diff
@@ -3,120 +3,145 @@
 Chapel Quickstart Instructions
 ==============================
 
-The following instructions are designed to help you build Chapel from
-source.  If you haven't already downloaded Chapel, obtain the latest
-release from http://chapel.cray.com/download.html, or refer to
-http://chapel.cray.com/install.html for other installation options.
+These instructions are designed to help you get started with
+a source distribution of Chapel.
 
 In the following instructions, note that building and using Chapel as
-described in steps 1-7 disables optional and advanced features in the
-interest of getting you a clean build as quickly as possible.  Step 8
-explains how to re-build in the preferred configuration with these
-additional features enabled.  Step 9 explains how to build Chapel for
-distributed memory execution.
+described in steps 2-5 disables optional and advanced features in the
+interest of getting you a clean build as quickly as possible. Later 
+sections explain how to re-build in the preferred configuration and how to
+enable more features, such as distributed memory execution.
 
 
 0) See `doc/prereqs.rst`_ for more information about system tools and
    packages you may need to have installed to build and run Chapel.
 
-.. _doc/prereqs.rst: http://chapel.cray.com/docs/1.13/usingchapel/prereqs.html
+1) If you don't already have Chapel 1.14, see
+   http://chapel.cray.com/install.html .
 
+2) If you are using a source release, build Chapel in a *quickstart*
+   configuration. The *quickstart* script used below is designed to help you
```

Comment:
I'm not convinced that making a single "script" with multiple commented commands is any  more helpful than a command with description per bullet item in the top-level .rst file.  Specifcally, I can see it raising questions for naive users like "So do I need to save this script into a file and run it or ..."?  I also think I'd rather read the comments in .rst rendered format rather than as comments in a code block.  Does rst have a second-level list such that these can be made steps a, b, c, d or somesuch if the goal is to minimize the number of top-level steps?